installing_on_kubernetes/_topics/migrate_from_appliance.md

```diff
@@ -0,0 +1,131 @@
+## Preparing the Kubernetes namespace
+The deploy directory referenced below can be found in [the manageiq-pods repo](https://github.com/ManageIQ/manageiq-pods/tree/master/manageiq-operator/deploy)
```

Comment:
This document is going to be a pain for productization... wondering how we can do that.  At a minimum perhaps we can move things like manageiq specific pod namings into https://github.com/ManageIQ/manageiq-documentation/blob/master/_data/product.yml (or use what's there?)